resources/template/redirect.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace PhpRedirect;
+
+class Redirect
+{
+    public function loc($uri)
+    {
+        header("location: $uri");
+    }
+}
+
+namespace JsRedirect;
+
+class Redirect
+{
+    public function loc($uri)
+    {
+        echo '<script> window.location.replace("' . $uri . '") </script>';
+    }
+    public function alert($msg)
+    {
+        echo '<script> alert(" ' . $msg . ' "); </script>';
+    }
+}
```

Comment:
Move this to two separate class files, something like Redirect/JsRedirect and Redirect/PhpRedirect

Reply:
Should I really move them to two separate class files?

Reply:
Yes either move them to two separate files, or keep them all in a single with 3 different methods.

Reply:
mmm okay, but I didn't get you!
they are inside a class so methods!
OR do you mean the alert method inside the JsRedirect?
you want me to create a different class for it?

Reply:
No i meant either: Keep one class per file Redirect/JsRedirect and Redirect/PhpRedirect. or if you want to keep a single file, then create a single redirect class having 3 methods like phLoc, jsLoc, jsAlert

Reply:
Okay got it